Fix javadoc in ErrorType

This is preventing the release 0.2.1

```
[ERROR] /usr/local/google/home/suztomo/cloud-opensource-java/dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ErrorType.java:34: error: malformed HTML
[ERROR]    * @see <a href="https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-5.html#jvms-5.4.3.4>Java
[ERROR]           ^
[ERROR] /usr/local/google/home/suztomo/cloud-opensource-java/dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ErrorType.java:34: error: bad use of '>'
[ERROR]    * @see <a href="https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-5.html#jvms-5.4.3.4>Java
[ERROR]                                                                                               ^
[ERROR] /usr/local/google/home/suztomo/cloud-opensource-java/dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ErrorType.java:35: error: unexpected end tag: </a>
[ERROR]    *     Virtual Machine Specification: 5.4.3.4. Interface Method Resolution</a>
[ERROR]                                                                             ^

```

--- a/dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ErrorType.java
+++ b/dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ErrorType.java
@@ -31,7 +31,7 @@ public enum ErrorType {
    *
    * @see <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-13.html#jls-13.4.9">Java
    *     Language Specification: 13.4.9. final Fields and static Constant Variables</a>
-   * @see <a href="https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-5.html#jvms-5.4.3.4>Java
+   * @see <a href="https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-5.html#jvms-5.4.3.4">Java
    *     Virtual Machine Specification: 5.4.3.4. Interface Method Resolution</a>
    */
   INCOMPATIBLE_CLASS_CHANGE("has changed incompatibly"),

--- a/kokoro/continuous.bat
+++ b/kokoro/continuous.bat
@@ -4,6 +4,6 @@ set JAVA_HOME=c:\program files\java\jdk1.8.0_152
 set PATH=%JAVA_HOME%\bin;%PATH%
 
 cd github/cloud-opensource-java
-mvn -B clean install
+mvn -B clean install javadoc:jar
 
 exit /b %ERRORLEVEL%

--- a/kokoro/continuous.sh
+++ b/kokoro/continuous.sh
@@ -7,4 +7,4 @@ set -x
 
 cd github/cloud-opensource-java
 # M2_HOME is not used since Maven 3.5.0 https://maven.apache.org/docs/3.5.0/release-notes.html
-mvn -B clean install
+mvn -B clean install javadoc:jar

